### PR TITLE
Fix sending messages to supergroups in Bot.send()

### PR DESCRIPTION
**Pull request by thesharp**

### CODE REVIEW

File: botogram/frozenbot.py [hunk 1]
```diff
@@ -276,12 +276,7 @@ def hide_commands(self):
     @utils.deprecated("Bot.%s()" % _proxy.__name__, "1.0",
                       "Use Bot.chat(id).%s() instead." % _proxy.__name__)
     def _wrapper(self, chat, *args, __proxy=_proxy, **kwargs):
-        # String chats are channels
-        if type(chat) == str:
-            obj = objects.Chat({"id": 0, "type": "channel", "username": chat},
-                               self.api)
-        else:
-            obj = objects.Chat({"id": chat, "type": ""}, self.api)
+        obj = self.api.call("getChat", {"chat_id": chat}, expect=objects.Chat)
```
Comment (pietroalbini) [inline botogram/frozenbot.py]: Couldn't this be `bot.chat(chat)`?

**Comment by thesharp** [comment]
Can't check right now if `bot.chat(chat)` will work there, but I took that `chat` from your old code and it works.

**Comment by pietroalbini** [comment]
Well, that's the [same code](https://github.com/pietroalbini/botogram/blob/6124bfcf386f1e0a8996e0e32080116b80d64251/botogram/frozenbot.py#L135) of `bot.chat()`, and it would be nice to don't duplicate code, because then I forgot to update it everywhere like this time ;-)

**Comment by thesharp** [comment]
Yeah, that would work.

**Comment by pietroalbini** [comment]
Cherry-picked the commit on the release branch and on master. Thank you!

**Pull request closed without merging**